### ISSUE TITLE
About Weak and Strong threshold values.

### ISSUE THREAD

**Issue by nysp78** [lorenmt/reco#33]
Hello again,
The values of weak(0.7) and strong(0.97) thresholds are used to filtered softmax ouputs or raw logits outputs? I'm saying this, because in the computation of the unsupervised loss I do not see any application of softmax. Are these thresholds probability values?

Thanks

**Comment by lorenmt** [comment]
Softmax probability output. It's probably clearer in the partial label setting.

https://github.com/lorenmt/reco/blob/71c1b7324fd36049c6657b122ba3fef373877b72/train_semisup_partial.py#L118-L122
